uwblueprint/planet-read
# Get Stories Available for Review by User

## Notion ticket link

[List all stories available to be reviewed by User](https://www.notion.so/uwblueprintexecs/List-all-stories-available-to-be-reviewed-by-User-40702ca6c0ef4daab38a9e2d997aed03)



## Implementation description
*  Created an additional story query that retrieves all of the stories available to be reviewed
*  Given a level and a language, query will return all story translations in that language that can be reviewed by someone with these credentials 



## Steps to test
1. Seed database with different stories and seed database with different story_translations that have a reviewer_id of None 
Example: 
`INSERT INTO stories (title, description, youtube_link, level, translated_languages) VALUES ('Kevin Eats Caviar for Lunch', 'He complains its too cheap', 'link', 4, '{"GERMAN", "ENGLISH_UK"}');`
`INSERT INTO stories (title, description, youtube_link, level, translated_languages) VALUES ('Kevin Eats Gold', 'He complains its too cheap', 'link', 2, '{"GERMAN", "MANDARIN"}');`
Insert into StoryTranslations: 
`INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (1,  'FRENCH', 'REVIEW', 1);`
`INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (2,  'ARABIC', 'REVIEW', 1);`
`INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (3,  'ARABIC', 'REVIEW', 1);`
`INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (2,  'GERMAN', 'REVIEW', 1);`
`INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (1,  'ENGLISH_US', 'START', 1);`
`INSERT INTO story_translations (story_id, language, stage, translator_id) VALUES (1,  'RUSSIAN', 'PUBLISH', 1);`

2. Call the query with various levels and languages and see returned list of story_translations


## What should reviewers focus on?
* Ensuring that the query returns story_translations as expected


## Checklist
